mbed-client-classic/m2mconnectionhandlerpimpl.h

```diff
@@ -58,7 +58,7 @@ class M2MConnectionHandlerPimpl {
     M2MConnectionHandlerPimpl(M2MConnectionHandler* base, M2MConnectionObserver &observer,
                               M2MConnectionSecurity* sec,
                               M2MInterface::BindingMode mode,
-                              M2MInterface::NetworkStack stack);
+                              M2MInterface::NetworkStack stack = M2MInterface::Uninitialized); //NetworkStack is deprecated
```

Comment:
This change (setting default value) should not be needed/done, as it makes API different than on other implementations of connectionhandler.